docs/MODELS.md

```diff
@@ -16,146 +13,138 @@ The below table will list all supported and unsupported functions of the patcher
-| MacBook2,1 | Late 2006 | ^^ | 32-Bit Firmware limitation |
-| MacBook3,1 | Late 2007 | ^^ | ^^ |
-| MacBook4,1 | Early 2008 | ^^ | - No GPU Acceleration in Mavericks and newer<br/>- No Keyboard and Trackpad<br/>- No USB |
-| MacBook5,1 | Late 2008 | <span style="color:#30BCD5"> YES </span> | - GPU Acceleration in Public Beta, see current issues ([#108](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/108))<br/> - UHCI/OHCI support in Public Beta, see current issues ([#1021](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/1021))|
-| MacBook5,2 | Early 2009 | ^^ | ^^ |
-| MacBook6,1 | Late 2009 | ^^ | ^^ |
-| MacBook7,1 | Mid-2010 | ^^ | ^^ |
-| MacBook8,1 | Mid-2015 | ^^ | <span style="color:green"> Everything is supported</span> |
-| MacBook9,1 | Early 2016 | ^^ | ^^ |
+| Model Name | Identifier | Tagged Issues | 
+| :--- | :--- | :--- | 
+| MacBook (13-inch, Late 2008) | `MacBook5,1` | [non-Metal GPU (macOS 11+)](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/108)<br>[USB 1.1 (macOS 13+)](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/1021)|
+| MacBook (13-inch, Early 2009) | `MacBook5,2` | ^^ | 
+| MacBook (13-inch, Late 2009) | `MacBook6,1` | ^^ | 
+| MacBook (13-inch, Mid 2010) | `MacBook7,1` | ^^ | 
+| MacBook (Retina, 12-inch, Early 2015) | `MacBook8,1` | [Legacy Metal (macOS 13+)](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/1008) | 
+| MacBook (Retina, 12-inch, Early 2016) | `MacBook9,1` | ^^ | 
+| MacBook (Retina, 12-inch, 2017) | `MacBook10,1` | Supported by Apple | 
 
 ### MacBook Air
 
-| SMBIOS | Year | Supported | Comment |
-| :--- | :--- | :--- | :--- |
-| MacBookAir1,1 | Early 2008 | <span style="color:red"> NO </span> | Requires SSE4.1 CPU |
-| MacBookAir2,1 | Late 2008 | <span style="color:#30BCD5"> YES </span> | - GPU Acceleration in Public Beta, see current issues ([#108](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/108))<br/> - UHCI/OHCI support in Public Beta, see current issues ([#1021](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/1021))|
-| MacBookAir3,1 | Late 2010 | ^^ | ^^ |
-| MacBookAir3,2 | ^^ | ^^ | ^^ |
-| MacBookAir4,1 | Mid-2011 | ^^ | - GPU Acceleration in Public Beta, see current issues ([#108](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/108))|
-| MacBookAir4,2 | ^^ | ^^ | ^^ |
-| MacBookAir5,1 | Mid-2012 | ^^ | <span style="color:green"> Everything is supported</span> |
-| MacBookAir5,2 | ^^ | ^^ | ^^ |
-| MacBookAir6,1 | Mid-2013, Early 2014 | ^^ | ^^ |
-| MacBookAir6,2 | ^^ | ^^ | ^^ |
-| MacBookAir7,1 | Early-2015, 2017 | ^^ | ^^ |
-| MacBookAir7,2 | ^^ | ^^ | ^^ |
+| Model Name | Identifier | Tagged Issues |
+| :--- | :--- | :--- |
+| MacBook Air (13-inch, Late 2008)<br>MacBook Air (13-inch, Mid 2009) | `MacBookAir2,1` | [non-Metal GPU (macOS 11+)](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/108)<br>[USB 1.1 (macOS 13+)](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/1021)| 
```

Comment:
Should use the `-` prefix to denote entries. Difficult to tell if the line is wrapping or it's a new entry
(this applies to the majority of entries on this page)